fontforge/cvundoes.c

```diff
@@ -1426,7 +1426,7 @@ return( copy(""));
 
     for ( lcnt = ly_fore; lcnt<dummy.layer_cnt; ++lcnt )
 	RefCharsFree(dummy.layers[lcnt].refs);
-    if ( dummy.layer_cnt!=2 )
```

Comment:
It's not a major issue, but I like to try to preserve as much useful information as possible from the reverse engineering required for each change set.